tests/contrib/utils/custom.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from datetime import datetime
+
+
+def today():
```

Comment:
I don't think we need a separate file for this, I think we can just inline this line in the test (also I don't think it's a contrib test?). Probably good to keep the call deterministic though (use a hardcoded execution date.

Reply:
This file was created only to import from a package not in the `PYTHONPATH` - probably does not require a new file since I could've imported anything. 
The test is deterministic, the date is not used for scheduling.

Reply:
Gotcha makes sense, let's add a comment explaining this.

For deterministic I meant more-so the output of the test. More of a nit but still.

Reply:
Also it's not a contrib test right? So this shouldn't be in contrib?

Reply:
We don't care about the output of the function, it's not checked since we're not testing the function. It's not a `contrib` test, so I will remove this file and import something from `tests/test_utils`